website/www/site/layouts/partials/section-menu/en/sdks.html

```diff
@@ -44,6 +44,7 @@
     <li><a href="/documentation/sdks/python-pipeline-dependencies/">Managing pipeline dependencies</a></li>
     <li><a href="/documentation/sdks/python-multi-language-pipelines/">Python multi-language pipelines quickstart</a></li>
     <li><a href="/documentation/sdks/python-unrecoverable-errors/">Python Unrecoverable Errors</a></li>
+    <li><a href="/documentation/sdks/python-sdk-image-build/">Python SDK image build</a></li>
```

Comment:
We have related content in:  https://beam.apache.org/documentation/runtime/environments/. I think it might be easier to find this link if we add a reference from that page.

We could also integrate (a portion of?) this  this content into that page, but having only  have a reference : for self-contained instructions on building a container image, follow this guide (link to new page), is ok too.

Reply:
I added a link to this guide in the environments page.